client/active-game/socket-handlers.js

```diff
@@ -14,12 +15,10 @@ export default function ({ siteSocket }) {
       payload: status,
     })
 
-    // TODO(tec27): Only invoke the lobby API for lobby loads, need to split this out for e.g.
-    // replays and AMM and such.
     if (status.state === 'playing') {
-      siteSocket.invoke('/lobbies/gameLoaded')
+      fetch('/api/1/games/' + encodeURIComponent(status.id), { method: 'put' })
     } else if (status.state === 'error') {
-      siteSocket.invoke('/lobbies/loadFailed')
+      fetch('/api/1/games/' + encodeURIComponent(status.id), { method: 'delete' })
```

Comment:
This feels weird semantically. With the way this works, it's like we're creating an entry for this ID if the game loads. But here, we're deleting an entity that was basically not created?

I think it'd feel less weird if the assumption was the server created this entity, and the `/gameLoaded` replacement here actually `PUT` some new data into that entry (e.g. `loaded: true` or something)